### PR TITLE
134  Another idea for smaller footer

### PR DESCRIPTION
**Pull request by ann-kilzer**
## Part of #134 

It's worth considering if we want to move the GitHub link into the Toolbar.

I also tried conditionally rendering the icons  to hide on 'xs', but worked for a bit and then the hamburger menu stopped working, almost as if something very wrong had happened in Vue. :(

I'm not particularly attached to the footer or bar being a certain way, and I quite like the look that @theyokohamalife came up with in #138 

Let me know your thoughts

### CODE REVIEW

File: components/Toolbar.vue [hunk 1]
```diff
@@ -31,6 +31,19 @@
...
+        <v-list-item
+          href="https://github.com/ourjapanlife/findadoc-frontend"
+          target="_blank"
+        >
+          <v-list-item-icon>
+            <v-icon>mdi-github</v-icon>
+          </v-list-item-icon>
+
+          <v-list-item-content>
+            <v-list-item-title>{{ $t("footer.github") }}</v-list-item-title>
```
Comment (ann-kilzer) [inline components/Toolbar.vue:43]: Eventually we should update this key name. But not tonight 😆

**Review by theyokohamalife — APPROVED** [review]
LGTM

**Comment by ann-kilzer** [comment]
Please merge after #138

File: components/Toolbar.vue (hunk 1 above)
Comment (ann-kilzer) [reply components/Toolbar.vue]: okay maybe tonight!!! ☕

**Pull request merged**